django_project/ta_system/admin.py

```diff
@@ -5,8 +5,9 @@
 from django.contrib.auth.models import User
 from django.core.exceptions import ObjectDoesNotExist
 from django.db import IntegrityError as AlreadyExistsError
-from django.shortcuts import render, redirect
-from django.urls import path
+from django.shortcuts import render, redirect, get_object_or_404
+from django.http import HttpResponse, HttpResponseRedirect
+from django.urls import path, reverse
```

Comment:
Are any of these new imports being used? I can't find references to them in the code you added.

Reply:
These imports were used for the change status functions, but I don't them anyway as I combined the functions into one. Will delete in the next commit